tachyon/zk/plonk/constraint_system/constraint_system.h

```diff
@@ -163,6 +163,12 @@ class ConstraintSystem {
     LookupPairs<std::unique_ptr<Expression<F>>> pairs =
         std::move(callback).Run(cells);
 
+    for (const LookupPair<std::unique_ptr<Expression<F>>>& pair : pairs) {
+      CHECK(!pair.input()->ContainsSimpleSelector())
```

Comment:
Nice! Next time I want to discuss within the team what kind of statement should be checked in release mode! Since I think it's one of the candidates to be checked in debug mode.

Reply:
I totally agree!